trade-finance-manager-api/src/v1/api.js

```diff
@@ -302,6 +302,32 @@ const updateDealCancellation = async ({ dealId, dealCancellationUpdate, auditDet
   }
 };
 
+/**
+ * Updates the deal cancellation object on a TFM AIN or MIN deal
```

Comment:
Do we need to mention AIN and MIN here, this function doesn't specifically care about this?